### DISCUSSION TITLE
Cyberpunk save deleted

### DISCUSSION THREAD

**Discussion by theunholy** [PhilipK/BoilR#153] [Q&A]
This issue is on my SteamDeck.

So a couple of days ago i set up boilr, it detected my games and it added images to my already made shortcuts, it also of course added a shortcut with image of my cyberpunk 2077 games installed in heroic launcher from gog. I tried the shirtcut it made 
and the game ran but steam would just stay in the game page but i could hear the game in the background running.  So i disregarded this and left the game as it was. Today i decided to retry this game to see id it would run properlly from the shortcut boilr made, but the same thing happened, and yes i updated boilr tomirs katest version today, so after i find out it does not run, i decided to delete the old shortcut that boilr made and create a new one, since boilr now seems to handle heroic game shorcuts better now(according to the changelog), and so i switched back to gameing mode and ran the game from the newlly created shortcut, to my suprise this time.it worked and launched correctlly, but when the game loaded something was not rightt greated me with the first installation message, and when i clicked load game my save files were gone. My question is does boilr mess with the proton containers in  order to handle heroic launcher games ? I reallly have no idea why boilr would delete ny cyberpunk 2077 save filesi searched all  of my proton cointainers  and indeed my save file are gone, the cyberpunk 2077 cointainer is still there but my saves are not.

**Comment by PhilipK** [comment]
I hope that it did not delete too long of a savegame?

Previously BoilR would try to launch the games by running the executable files directly. Now it launches Heroic with a launch parameter that tells Heroic to run the game instead. I don't know how Cyberpunk handles the savegames, but that might make a difference?

When you create a shortcut and set it to use proton, steam creates a folder in ~./steam/steam/steamapps/compatdata and I think that your savegame is in there somewhere.
If you deleted it, and then made a new shortcut, steam will have created a new folder.

**Comment by theunholy** [comment]
Yes you are correct , with the new launch method it created a new proton container, so i started searching and low and behold it seems heroic launcher had its own save game folder, and yes my saves were there!,  so i just moved my saves to new proton container, and all was good again, thank you for this awesome app, cyberpunk is breeze now to launch
